UploadDaemon/SymbolAnalysis/FileCoverage.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UploadDaemon.SymbolAnalysis
+{
+    /// <summary>
+    /// The line coverage collected for one file.
+    /// </summary>
+    public class FileCoverage
+    {
+        /// <summary>
+        /// The ranges of inclusive start and end lines that are covered in the file.
+        /// </summary>
+        public HashSet<(uint, uint)> CoveredLineRanges { get; } = new HashSet<(uint, uint)>();
+
+        public FileCoverage()
```

Comment:
Remove empty constructor.

Reply:
cool! I thought I needed that ;-) good catch